pgcli/main.py

```diff
@@ -520,8 +522,18 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
             '\thost: %r'
             '\tport: %r', database, user, host, port)
 
+    obscure_process_password()
     pgcli.run_cli()
 
+def obscure_process_password():
+    process_title = setproctitle.getproctitle()
+    if '://' in process_title:
+        process_title = re.sub(r":(.*):(.*)@", r":\1:xxxx@", process_title)
+    elif "=" in process_title:
+        process_title = re.sub(r"password=\S+", "password=xxxx", process_title)
```

Comment:
This will break if the password has spaces in them. I tried it by changing my password locally with spaces in them and here's the output: 

```
amjith          67478   0.1  0.2  2499852  27260 s010  S+   11:10PM   0:00.34 /Users/amjith/.virtualenvs/pgcli/bin/python2.7 /Users/amjith/.virtualenvs/pgcli/bin/pgcli host=localhost user=amjith password=xxxx check'
```

I had set the password as `amjith check` the first portion was masked and the portion after the space is left alone.